libsolidity/formal/SMTEncoder.cpp

```diff
@@ -1135,11 +1109,8 @@ bool SMTEncoder::visit(MemberAccess const& _memberAccess)
 				);
 		}
 		else
-			m_errorReporter.warning(
-				9551_error,
-				_memberAccess.location(),
-				"Assertion checker does not yet support this expression."
-			);
+			solAssert(false, "");
```

Comment:
I wonder if these should be `solUnimplementedAssert`?

Reply:
Yea, that makes sense. I don't know what could go in that branch exactly, but I also don't know that it can't.

Reply:
I didn't mean this branch specifically, but in general.